spec/testing_itself.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'iron_trail/testing/rspec'
+
+IronTrail::Testing.disable!
```

Comment:
suggestion: also add a test where it is enabled by default, to ensure that `iron_trail: false` is working properly

Reply:
@peterbrendel done in https://github.com/trusted/iron_trail/pull/9/commits/dd2709679ba1049991875300ab8b471901ec288c